src/pattern-library/components/patterns/data/DataTablePage.tsx

```diff
@@ -1,17 +1,87 @@
-import { useState } from 'preact/hooks';
+import { useCallback, useMemo, useState } from 'preact/hooks';
 
-import { DataTable, Scroll } from '../../../../';
+import { DataTable, type DataTableProps, Scroll } from '../../../../';
+import type { Order } from '../../../../components/data/DataTable';
 import Library from '../../Library';
 import { nabokovNovels } from '../samples';
 import type { NabokovNovel } from '../samples';
 
 const nabokovRows = nabokovNovels();
 const nabokovColumns = [
-  { field: 'title', label: 'Title' },
-  { field: 'year', label: 'Year' },
-  { field: 'language', label: 'Language' },
+  { field: 'title' as const, label: 'Title' },
+  { field: 'year' as const, label: 'Year' },
+  { field: 'language' as const, label: 'Language' },
 ];
 
+type StrictNabokovNovel = Omit<NabokovNovel, 'translatedTitle'>;
```

Comment:
Why `Strict`?

Reply:
Nah, because I couldn't come up with a better wording 😅. I just wanted to exclude `translatedTitle` because it's optional, hence the word "Strict".

I guess we could name it `BasicNabokovNovel`, `BaseNabokovNovel`, `SimpleNabokovNovel`, or something in those lines.